bindings/python/src/encoding.rs

```diff
@@ -263,7 +263,7 @@ impl PyEncoding {
     ///
     /// Returns:
     ///     :obj:`Tuple[int, int]`: The range of tokens: :obj:`(first, last + 1)`
-    #[args(sequence_index = 0)]
+    #[pyo3(signature = (word_index, sequence_index = 0))]
     #[pyo3(text_signature = "(self, word_index, sequence_index=0)")]
```

Comment:
Does the inspected signature automatically include `self` here too ? If yes then yes let's remove those `text_signature`.

There were other things to check for, but as long as the `.pyi` are consistent I'll agree to remove them (especially if they are duplicate, and the pyo3 signature is actually validated)

Reply:
You already answered this :)